lib/http_server_fixture/data/test-data.json

```diff
@@ -1 +1 @@
-{}
\ No newline at end of file
+{"1":{"task":"a new task"},"2":{"task":"a new task"},"3":{"task":"a new task"}}
```

Comment:
is this missing a new-line at he end of the file? I keep seeing this in the PRs.  Maybe now is the time to fix it?

Reply:
I agree! I might open a quick PR after this one to set up test teardowns so that after the test is run the test-data files are deleted. It's fairly simple to execute for the ExUnit tests, not entirely sure how I can make that work for the integration tests...